data retrieval/src/main/java/de/tinf15b4/kino/retrieval/scraper/AbstractCinemaScraper.java

```diff
@@ -40,39 +40,46 @@
 
     public AbstractCinemaScraper() {
         tmdb = new TmdbDataRetriever();
+        cinema = new Cinema();
+        cinema.setName("UNKNOWN");
```

Comment:
Problem is that you use the web driver in zkm scraper in the getCinema method, so the driver has to be initialized before. But then we have the problem that initializing the driver uses the cinema name (its just for logging, but I didnt feel like inserting null checks at a thousand lines of code). This way the first few logging entries are unknown, but after the cinema is set correctly there are no problems anymore. 

Nevertheless your right and I will insert a null check after the getCinema method. If it is null we will throw an exception instead of continuing with unknown.